{"name": "test_loginTestsOne", "status": "broken", "statusDetails": {"message": "AttributeError: 'Test_TestOne' object has no attribute 'customer1'", "trace": "self = <Tests.test_login.Test_TestOne object at 0x00000294A8A67DD0>, setup = <selenium.webdriver.edge.webdriver.WebDriver (session=\"e7be6f651defdb5ab248ddbfe2195c2a\")>\n\n    @pytest.mark.consy\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_loginTestsOne(self, setup):\n        self.driver = setup\n        self.driver.get(self.bankingURL)\n        self.driver.maximize_window()\n        self.login = LoginPage(self.driver)\n        self.login.clickCustomerLoginButton()\n>       self.login.enterCustomerName(self.customer1)\nE       AttributeError: 'Test_TestOne' object has no attribute 'customer1'\n\nTests\\test_login.py:20: AttributeError"}, "start": 1729617834867, "stop": 1729617852889, "uuid": "bbf201c8-8d7a-41cd-af09-828caf561d1b", "historyId": "16354c69e511ecc93a3c1ceee382e6a5", "testCaseId": "16354c69e511ecc93a3c1ceee382e6a5", "fullName": "Tests.test_login.Test_TestOne#test_loginTestsOne", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "consy"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_TestOne"}, {"name": "host", "value": "ZAJ-TUTJACONS"}, {"name": "thread", "value": "31624-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login"}]}